fcrepo-search-api/src/main/java/org/fcrepo/search/api/SearchResult.java

```diff
@@ -29,15 +28,25 @@
  */
 public class SearchResult {
     @JsonProperty
-    private PaginationInfo pagination = new PaginationInfo();
+    private PaginationInfo pagination;
     @JsonProperty
-    private List<Map<String, Object>> items = new ArrayList();
+    private List<Map<String, Object>> items;
 
     /**
-     * Default constructor
+     * Default Constructor
      */
-    public SearchResult() {
-
+    public SearchResult() {}
```

Comment:
An empty default public constructor is implicit. This can be removed.

Reply:
Required for json marshalling.